analyzer/src/test/kotlin/managers/SpdxDocumentFileTest.kt

```diff
@@ -187,7 +187,7 @@ class SpdxDocumentFileTest : WordSpec({
             )
 
             val spdxPackageId = "SPDXRef-Package-zlib"
-            val spdxPackage = externalDocumentReference.getSpdxPackage(spdxPackageId, File(""))
+            val spdxPackage = externalDocumentReference.getSpdxPackage(spdxPackageId, File(""), mutableListOf())
```

Comment:
It would be nice to have a test for this new functionality.

Reply:
Filling issues is now tested, in the run of replacing exceptions thrown with issues being filed.